licenses/third_party.md

```diff
@@ -20,7 +20,6 @@ please keep this up to date with every new library use.
 | DPDK            | BSD                                |
 | fmt             | BSD                                |
 | gmp             | LGPL v3                            |
-| gnutls          | LGPL v2.1                          |
```

Comment:
should nettle, gmp, etc... also be removed from this list?

Reply:
Thanks, yup